modules/serverless-tech-preview-features.adoc

```diff
@@ -13,30 +13,33 @@ The following table provides information about which {ServerlessProductName} fea
 .Generally Available and Technology Preview features tracker
 [cols="2,1,1,1",options="header"]
 |====
-|Feature |1.32|1.33|1.34
```

Comment:
The "Go functions using S2I builder" should be marked as GA in this table as mention GA for this feature above.